src/application.py

```diff
@@ -244,15 +249,14 @@ def do_activate(self):
             ui.enable_axes_actions(self, None, self)
             window.present()
 
-    def set_entry_css(self, entry, _string, button):
+    def set_entry_css(self, _object, _param, entry, button):
         try:
             value = utilities.string_to_float(entry.get_text())
-            if value is None and entry.get_text() != "":
-                entry.add_css_class("error")
-                button.set_sensitive(False)
+            if value is not None and entry.get_text() != "":
```

Comment:
```suggestion
            if value is not None:
```
If the entry is a blank string, the value will be zero. Not completely sure why I had I had that as condition, but probably has to do with the order I was testing exceptions. (First handled empty strings, then noticed that other strings could also deliver `None` without throwing an error so added `None` as condition.)

Reply:
I'm not sure that would be the better solution. An empty string passing as 0 wouldn't be intuitive. So the idea here being still, that you cant use the button when the input is empty

Reply:
If `entry.get_text() != ""`, then value will always return `None`. Thus the second condition seems kind of redundant, as if the second condition holds true (`entry.get_text() != ""`), then the first one (`value is not None`) will be false. So it would never hit that case, and removing the second condition would functionally be the exact same thing.

Unless I'm missing something. My point was not to change anything with the functional logic itself, that is fine. It works exactly as it should work now imo, just meant there's a logical redundancy.